x-pack/plugins/observability_solution/observability/public/pages/alert_details/components/page_title.stories.tsx

```diff
@@ -10,34 +10,36 @@ import { ComponentStory } from '@storybook/react';
 import { EuiPageTemplate } from '@elastic/eui';
 import { ALERT_RULE_CATEGORY } from '@kbn/rule-data-utils';
 
-import { PageTitle as Component, PageTitleProps } from './page_title';
+import { PageTitleContent as Component, PageTitleContentProps } from './page_title_content';
 import { alert } from '../mock/alert';
 
 export default {
   component: Component,
-  title: 'app/AlertDetails/PageTitle',
+  title: 'app/AlertDetails/PageTitleContent',
   alert,
 };
 
-const Template: ComponentStory<typeof Component> = (props: PageTitleProps) => (
+const Template: ComponentStory<typeof Component> = (props: PageTitleContentProps) => (
   <Component {...props} />
 );
 
-const TemplateWithPageTemplate: ComponentStory<typeof Component> = (props: PageTitleProps) => (
+const TemplateWithPageTemplate: ComponentStory<typeof Component> = (
```

Comment:
Some of the stories in this page does not make sense anymore ([storybook](https://ci-artifacts.kibana.dev/storybooks/pr-193958/bb7e9e9fec60146559c15d41886f51e053034921/observability/index.html?path=/story/app-alertdetails-pagetitlecontent--page-title-content))

Is it possible to bring back the title? If not, we can remove the `Page Title Content for Anomaly` as it is exactly the same as `Page Title Content`

Reply:
I'm not sure about moving the `Title` back. Let's remove `Page Title Content for Anomaly`